source/compatibility.tex

```diff
@@ -1881,6 +1881,19 @@
 Valid \CppIII{} code that attempts to explicitly instantiate a container using
 a user-defined type with no default constructor may fail to compile.
 
+\diffref{container.requirements.general}
+\change
+Containers now access their allocators through the \tcode{allocator_traits}
+template.
+\rationale
+Simplifies writing new allocators.
+\effect
+\tcode{allocator_traits} supplies default definitions for many allocator type
+names and operations. Valid \CppIII{} code that follows the original
+allocator-aware container requirements may not support allocators written to
```

Comment:
There were no "allocator-aware container requirements" before C++11, so it's unclear what "the original" ones are. We did have Allocator requirements in C++03, and those are what changed dramatically.

Maybe it would help to have an example showing a function template with an `Allocator` parameter, assuming the presence of `pointer` and `construct` members, which need to be accessed via `allocator_traits` since C++11.